samtranslator/model/api/api_generator.py

```diff
@@ -626,14 +681,21 @@ def _construct_alias_target(self, domain: Dict[str, Any], api_domain_name: str,
         return alias_target
 
     @cw_timer(prefix="Generator", name="Api")
-    def to_cloudformation(self, redeploy_restapi_parameters, route53_record_set_groups):  # type: ignore[no-untyped-def]
+    def to_cloudformation(
+        self, redeploy_restapi_parameters: Optional[Any], route53_record_set_groups: Dict[str, Route53RecordSetGroup]
+    ) -> List[Any]:
         """Generates CloudFormation resources from a SAM API resource
 
         :returns: a tuple containing the RestApi, Deployment, and Stage for an empty Api.
         :rtype: tuple
         """
         rest_api = self._construct_rest_api()
-        domain, basepath_mapping, route53 = self._construct_api_domain(rest_api, route53_record_set_groups)
+        apiDomainResponse = self._construct_api_domain(rest_api, route53_record_set_groups)
+        domain = apiDomainResponse.Domain
+        basepath_mapping = apiDomainResponse.ApiGWBasePathMappingList
+        route53_recordsetGroup = apiDomainResponse.RecordSetsGroup
```

Comment:
Mixing both snake case and camel case. 😩 Let's use `snake_case` everywhere.

Reply:
lol updated